.github/workflows/main.yml

```diff
@@ -310,7 +310,7 @@ jobs:
     uses: ./.github/workflows/build-windows.yml
     with:
       platform: windows-x64
-      msvc-toolset-version: '14.29'
+      msvc-toolset-version: '14.43'
```

Comment:
I wish I understood why `14.43`, since I don't see any reference to it in current runner docs: https://github.com/actions/runner-images/blob/main/images/windows/Windows2025-Readme.md. But I see you have tried other versions, and this one works?

From https://learn.microsoft.com/en-us/lifecycle/faq/visual-c-faq it sounds like `14.43` is not the final one. ( Compare with `14.29` for VS 2019.) It is not even listed as "supported"... `14.42` is listed as supported until July 14, 2026. I see you have tried `14.42`: https://github.com/openjdk/jdk/pull/25628/commits/6f7922f5c70e4ffef82005b3ea702673b589eb64 -- did it not work?